CollAction/Services/FestivalService.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Microsoft.Extensions.Options;
+
+namespace CollAction.Services
+{
+    public class FestivalService : IFestivalService
+    {
+        private readonly FestivalServiceOptions _festivalOptions;
+        private readonly DateTime DefaultEndDate = new DateTime(2018, 5, 27, 23, 59, 59);
```

Comment:
The default enddate seems unnecessary.

I'd say if there is no date date, don't show the festival stuff. No defaults necessary